providers/aws/services/vpc/vpc_service_test.py

```diff
@@ -0,0 +1,254 @@
+        # main_route_table = ec2_resource.RouteTable(main_route_table_id)
+        # main_route_table.create_route(
+        #     DestinationCidrBlock="10.0.0.4/24", VpcPeeringConnectionId=vpc_pcx_id
+        # )
+
+        # VPC client for this test class
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        vpc.vpc_peering_connections[0].route_tables = [
+            Route(
+                main_route_table_id,
+                ["10.0.0.4/24"],
+            )
+        ]
+        assert len(vpc.vpc_peering_connections[0].route_tables) == 1
+
+    # Test VPC Describe VPC Endpoints
+    @mock_ec2
+    def test__describe_vpc_endpoints__(self):
+        # Generate VPC Client
+        ec2_client = client("ec2", region_name=AWS_REGION)
+        # Create VPC endpoint
+        vpc = ec2_client.create_vpc(CidrBlock="10.0.0.0/16")["Vpc"]
+
+        route_table = ec2_client.create_route_table(VpcId=vpc["VpcId"])["RouteTable"]
+        ec2_client.create_vpc_endpoint(
+            VpcId=vpc["VpcId"],
+            ServiceName="com.amazonaws.us-east-1.s3",
+            RouteTableIds=[route_table["RouteTableId"]],
+            VpcEndpointType="Gateway",
+            PolicyDocument=json.dumps(
+                {
+                    "Statement": [
+                        {
+                            "Action": "*",
+                            "Effect": "Allow",
+                            "Principal": "*",
+                            "Resource": "*",
+                        }
+                    ]
+                }
+            ),
+        )
+        # VPC client for this test class
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        assert len(vpc.vpc_endpoints) == 1
+
+    # Test VPC Describe VPC Endpoint Services
+    @mock_ec2
+    @mock_elbv2
+    def test__describe_vpc_endpoint_services__(self):
+        # Generate VPC Client
+        ec2_client = client("ec2", region_name=AWS_REGION)
+        elbv2_client = client("elbv2", region_name=AWS_REGION)
+
+        vpc = ec2_client.create_vpc(
+            CidrBlock="172.28.7.0/24", InstanceTenancy="default"
+        )
+        subnet = ec2_client.create_subnet(
+            VpcId=vpc["Vpc"]["VpcId"],
+            CidrBlock="172.28.7.192/26",
+            AvailabilityZone=f"{AWS_REGION}a",
+        )
+        lb_name = "lb_vpce-test"
+        lb_arn = elbv2_client.create_load_balancer(
+            Name=lb_name,
+            Subnets=[subnet["Subnet"]["SubnetId"]],
+            Scheme="internal",
+            Type="network",
+        )["LoadBalancers"][0]["LoadBalancerArn"]
+
+        service = ec2_client.create_vpc_endpoint_service_configuration(
+            NetworkLoadBalancerArns=[lb_arn]
+        )
+        # VPC client for this test class
+        audit_info = self.set_mocked_audit_info()
+        vpc = VPC(audit_info)
+        assert (
+            len(vpc.vpc_endpoint_services) == 0
```

Comment:
Also check that the values created are the same that the received in the client

Reply:
We have to wait for the issue to be fixed.

Reply:
Include an issue in the repo please to keep track of it